focal/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN add-apt-repository --yes "ppa:edd/r-4.0" \
 	&& apt-get install -y --no-install-recommends \
                 r-cran-bspm \
         && echo "suppressMessages(bspm::enable())" >> /etc/R/Rprofile.site \
-        && echo "options(pkgType=\"binary-source\")" >> /etc/R/Rprofile.site \
+        && echo "options(bspm.version.check=FALSE)" >> /etc/R/Rprofile.site \
```

Comment:
Hm, to a first approximation I liked what we riffed on yesterday (`options(bspm.fast=TRUE)`) better: "it adds something", namely "fast" mode.  Whereas `bspm.version.check=FALSE` seems to take something away...

The help text is fine though.

Reply:
Strictly speaking, it does _take something away_, which is version checks and all the code dedicated to determine what would be the best combination of binary and source packages depending on the user preference, and whether to install soft dependencies too, now that the `dependencies` argument is considered in the mix.

Also "fast" was mainly an internal term in our discussions, so it was harder to explain and document for a general audience. Plus there was already a `bspm.backend.check` option, so this follows that naming. I've added a dedicated manual page for all the options supported, `bspm-options`.